schemas/com.snowplowanalytics.snowplow.storage.rdbshredder/output/jsonschema/1-0-0

```diff
@@ -0,0 +1,68 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for information about the shredded data, for loader to read",
+    "self": {
+      "vendor": "com.snowplowanalytics.snowplow.storage.rdbshredder",
+      "name": "output",
+      "format": "jsonschema",
+      "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+      "s3_path" : {
+        "description": "S3 path containing shredded data",
+        "type": "string",
+        "minLength": 10
+      },
+      "min_timestamp": {
+        "description": "Minimum timestamp in shredded data",
+        "type": "string",
+        "format": "date-time"
+      },
+      "max_timestamp": {
+        "description": "Maximum timestamp in shredded data",
+        "type": "string",
+        "format": "date-time"
+      },
```

Comment:
I think that `s3_path`, `min_timestamp` and `max_timestamp` should be with each element of the array, so in the end we would have nothing else than an array of shredded types